<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestaurantsView</class>
 <widget class="QWidget" name="RestaurantsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_restList" native="true">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>0</y>
     <width>191</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_menuList" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>440</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_menu">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>440</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#label_menu
{
	color: black;
	background-color: #7F7F7F;
	font-size: 28px;
	font: bold italic;
}</string>
   </property>
   <property name="text">
    <string>MENU</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_restName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#label_restName
{
	background-color: #303030;
	font-size: 46px;
	color: white;
	font:bold italic;
}</string>
   </property>
   <property name="text">
    <string>Restaurant</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget_background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_background {
background: url(:/res/artboard_purple.png);
}</string>
   </property>
  </widget>
  <zorder>widget_background</zorder>
  <zorder>widget_restList</zorder>
  <zorder>widget_menuList</zorder>
  <zorder>label_menu</zorder>
  <zorder>label_restName</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
